### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aziya-im**
I have changed the date for deprecation of version 6

**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The schedule for the current maintained branches is:
 
 |  Plugin Major Version   |  Release Date  |  Level 1 Support ends   |  Level 2 Support ends   |  Level 3 Support ends  |
 |  :-----          |  :-----          |  :-----          |  :-----          |  :-----          |
-|  Version 6 |  March 2020  |  September 31 2021 |  March 31 2022 |  September 31 2022 |
+|  Version 6 |  March 2020  |  September 31 2021 |  March 31 2022 |  December 31 2022 |
 |  Version 7 |  April 2021 |  September 31 2022 |  March 31 2023 |  September 31 2023 |
 |  Version 8 |  Novemeber 2021 |  March 31 2023 |  September 2023 | March 2024 |
```
Comment (acampos1916) [inline README.md:35]:
```suggestion
|  Version 8 |  November 2021 |  March 31 2023 |  September 2023 | March 2024 |
```

**Review by acampos1916 — DISMISSED** [review]

**Review by michaelpaul — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The schedule for the current maintained branches is:
 
 |  Plugin Major Version   |  Release Date  |  Level 1 Support ends   |  Level 2 Support ends   |  Level 3 Support ends  |
 |  :-----          |  :-----          |  :-----          |  :-----          |  :-----          |
-|  Version 6 |  March 2020  |  September 31 2021 |  March 31 2022 |  September 31 2022 |
+|  Version 6 |  March 2020  |  September 31 2021 |  March 31 2022 |  December 31 2022 |
```
[the remark this answers is not in the document]
Comment (acampos1916) [reply README.md]: @aziya-im have you seen this suggestion from Alessio? I believe we had September for Lvl3 v6 intentionally

File: README.md [hunk 3]
```diff
@@ -30,9 +37,9 @@ The schedule for the current maintained branches is:
 
 |  Plugin Major Version   |  Release Date  |  Level 1 Support ends   |  Level 2 Support ends   |  Level 3 Support ends  |
 |  :-----          |  :-----          |  :-----          |  :-----          |  :-----          |
-|  Version 6 |  March 2020  |  September 31 2021 |  March 31 2022 |  September 31 2022 |
+|  Version 6 |  March 2020  |  September 30 2021 |  March 31 2022 |  December 31 2022 |
 |  Version 7 |  April 2021 |  September 31 2022 |  March 31 2023 |  September 31 2023 |
```
Comment (Aleffio) [inline README.md:41]:
```suggestion
|  Version 7 |  April 2021 |  September 30 2022 |  March 31 2023 |  September 30 2023 |
```

File: README.md [hunk 4]
```diff
@@ -17,6 +17,13 @@ This plugin supports Magento2 version
 * 2.3.7 and higher
 * 2.4 and higher
 
+# Releases
```
Comment (michaelpaul) [inline README.md:20]: Use a subheading for this section: 
```suggestion
## Releases
```

**Review by michaelpaul — APPROVED** [review]

**Review by acampos1916 — APPROVED** [review]

**Review by candemiralp — APPROVED** [review]

**Pull request merged**